woostarter/patterns/about-4-column-features-row-icons.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Title: 4-column features row with icons
+ * Slug: woostarter/store-features-row-icons-4-columns
+ * Categories: text, about
+ * Keywords: about, text
+ * Description: A four-column section with store featured services.
+ *
+
+?>
+<!-- wp:group {"metadata":{"name":"Small store features row with icons"},"align":"full","style":{"spacing":{"padding":{"top":"0","bottom":"0","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"blockGap":"0","margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="margin-top:0;margin-bottom:0;padding-top:0;padding-right:var(--wp--preset--spacing--50);padding-bottom:0;padding-left:var(--wp--preset--spacing--50)"><!-- wp:spacer {"height":"var:preset|spacing|60"} -->
+<div style="height:var(--wp--preset--spacing--60)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"align":"wide","style":{"spacing":{"blockGap":{"top":"var:preset|spacing|70"}}}} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"verticalAlignment":"center"} -->
+<div class="wp-block-column is-vertically-aligned-center"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
+<div class="wp-block-group"><!-- wp:image {"id":479,"width":"24px","sizeSlug":"full","linkDestination":"none","align":"center"} -->
+<figure class="wp-block-image aligncenter size-full is-resized"><img src="https://kiri.mystagingwebsite.com/wp-content/uploads/2025/02/icon-truck.webp" alt="Truck icon." class="wp-image-479" style="width:24px"/></figure>
```

Comment:
This `src` attribute needs to be updated to `<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/icon-truck.webp` before merging.